tests/unit/test_models.py

```diff
@@ -87,62 +87,70 @@ def test_StreamingBody_name():
     body = models.StreamingBody(r)
 
     assert body.name.startswith('planet-')
-    assert (body.name.endswith('.tiff') or
-            body.name.endswith('.tif'))
-
-
-@pytest.mark.parametrize('headers,expected', [
-    ({
-        'date': 'Thu, 14 Feb 2019 16:13:26 GMT',
-        'last-modified': 'Wed, 22 Nov 2017 17:22:31 GMT',
-        'accept-ranges': 'bytes',
-        'content-type': 'image/tiff',
-        'content-length': '57350256',
-        'content-disposition': 'attachment; filename="open_california.tif"'
-    }, 'open_california.tif'),
-    ({
-        'date': 'Thu, 14 Feb 2019 16:13:26 GMT',
-        'last-modified': 'Wed, 22 Nov 2017 17:22:31 GMT',
-        'accept-ranges': 'bytes',
-        'content-type': 'image/tiff',
-        'content-length': '57350256'
-    }, None),
-    ({}, None)
-])
+    assert (body.name.endswith('.tiff') or body.name.endswith('.tif'))
+
+
+NO_NAME_HEADERS = {
+    'date': 'Thu, 14 Feb 2019 16:13:26 GMT',
+    'last-modified': 'Wed, 22 Nov 2017 17:22:31 GMT',
+    'accept-ranges': 'bytes',
+    'content-type': 'image/tiff',
+    'content-length': '57350256'
+}
+OPEN_CALIFORNIA_HEADERS = {
+    'date': 'Thu, 14 Feb 2019 16:13:26 GMT',
+    'last-modified': 'Wed, 22 Nov 2017 17:22:31 GMT',
+    'accept-ranges': 'bytes',
+    'content-type': 'image/tiff',
+    'content-length': '57350256',
+    'content-disposition': 'attachment; filename="open_california.tif"'
+}
+
+
+@pytest.mark.parametrize('headers,expected',
+                         [(OPEN_CALIFORNIA_HEADERS, 'open_california.tif'),
+                          (NO_NAME_HEADERS, None),
+                          ({}, None)])  # yapf: disable
```

Comment:
I think it helps comprehension here to have each 'test' on its own line. yapf would like to join the last two lines. I couldn't find a way around it, so just disabled it.

Reply:
FWIW, black would put them each on their own line if the list's text was wider than  some value. See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines.

Reply:
Well, I'm with black on this one. 😆